Assets/Scenes/scene.unity

```diff
@@ -1116,7 +1116,7 @@ Light:
   m_BounceIntensity: 1
   m_ColorTemperature: 6570
   m_UseColorTemperature: 0
-  m_BoundingSphereOverride: {x: 0, y: 0, z: 0, w: 0}
+  m_BoundingSphereOverride: {x: -1, y: 10, z: 11, w: -12}
```

Comment:
Hmm I'm not sure what I did to incur this change. @croconut do you think I should just manual reset this line?

Reply:
i think you need to set useBoundingSphereOverride to see a difference anyways

Reply:
Okay, I've gone ahead and reverted this in any case